python/pyspark/sql/context.py

```diff
@@ -63,7 +63,33 @@ def toDF(self, schema=None, sampleRatio=None):
         """
         return sqlContext.createDataFrame(self, schema, sampleRatio)
 
+    def schema(self, datatype):
```

Comment:
The difference is: this `schema` method support parsing schema string, and guarantee no type inference. It also supports RDD of custom objects, and use a flat schema here, and it will wrap the custom object into a row automatically.

Is it worth to add this new method?